web/themes/contrib/civictheme/includes/utilities.inc

```diff
@@ -120,6 +120,19 @@ function civictheme_get_layout_builder_settings_per_view_mode(string $entity_typ
  * @SuppressWarnings(PHPMD.StaticAccess)
  */
 function civictheme_media_get_variables(MediaInterface $media): ?array {
+  $variables = [
+    'media_name' => t('@name', ['@name' => $media->label()]),
+    'created' => civictheme_format_datetime((int) $media->getCreatedTime(), 'civictheme_short_date'),
+    'changed' => civictheme_format_datetime($media->getChangedTime(), 'civictheme_short_date'),
+  ];
+
+  if ($media->bundle() === 'civictheme_remote_video') {
```

Comment:
We can make it more extensible with:
```suggestion
  if ($media->getSource() instanceof \Drupal\media\Plugin\media\Source\OEmbed) {
```

Reply:
Also updated to use $source->getMetadata($media, 'url')
and not specific 'field_c_m_oembed_video' field.

Reference 
![Screenshot 2024-11-19 at 8 53 46 AM](https://github.com/user-attachments/assets/e42e1a51-9556-447f-9aeb-d6ac783ce869)